pkg/commands/compute/init.go

```diff
@@ -234,6 +256,7 @@ func (c *InitCommand) Exec(in io.Reader, out io.Writer) (err error) {
 	// that isn't natively supported by the platform.
 	if c.CloneFrom != "" {
 		if !isExistingService {
+			fmt.Printf("Fetching package from: %s\n", c.CloneFrom)
```

Comment:
Two general notes to make...

1. We should use `text.Output` or `text.Info` for these messages as it means we can pass in mocked stdout (that is passed in as the `out` argument to the `Exec()` method.

2. We should probably wrap this in a check for the verbose flag (as some users would consider this unnecessary noise).

Otherwise, we already inform the user of this information here:
https://github.com/fastly/cli/blob/8912844aa6dc0aa64f790b5ec40f78fe4c886540/pkg/commands/compute/init.go#L82-L87

Reply:
This was also leftover debugging, honestly 😓 I'll do another pass to make sure no more of that stayed in